commit 6a103e87cb3f1d0df4ca2cd9dc788a8b70213b6f (HEAD -> fix-markers-in-numbered-lists, fork/fix-markers-in-numbered-lists)
Author: Mike Kazantsev <mk.fraggod@gmail.com>
Date:   Wed Aug 12 17:47:28 2026 +0500

    fix: Epub - render markers in <ol> numbered lists
    
    Currently such lists look like text paragraphs, where it's impossible to tell
    multi-paragraph items apart.
    This at least marks items somehow, even if not exactly how they're supposed to be.

See also https://github.com/obijuankenobiii/inx/pull/118

diff --git a/lib/Epub/Epub/parsers/ChapterHtmlSlimParser.cpp b/lib/Epub/Epub/parsers/ChapterHtmlSlimParser.cpp
index cc6cd3c..a77af6e 100644
--- a/lib/Epub/Epub/parsers/ChapterHtmlSlimParser.cpp
+++ b/lib/Epub/Epub/parsers/ChapterHtmlSlimParser.cpp
@@ -1883,8 +1883,6 @@ void XMLCALL ChapterHtmlSlimParser::startElement(void* userData, const XML_Char*
       }
       if (tagLower == "ul" || tagLower == "ol") {
         self->listNoIndentDepths_.push_back(self->depth);
-      }
-      if (tagLower == "ul") {
         self->ulBulletVisibleStack.push_back(!self->css().isListStyleNone(tagLower, classAttr, idAttr, styleAttr));
         self->ulBulletVisibleDepths.push_back(self->depth);
       } else if (tagLower == "li") {
